test/test_puma_server_ssl.rb

```diff
@@ -55,6 +57,7 @@ def start_server
     if Puma.jruby?
       ctx.keystore =  File.expand_path "../../examples/puma/keystore.jks", __FILE__
       ctx.keystore_pass = 'blahblah'
+      ctx.alias = 'mydomain'
```

Comment:
I think we should add something about this in the HISTORY ("`alias must be provided`")